### ISSUE TITLE
Can I use your model to train or infrence trajectory on KITTI dataset?

### ISSUE THREAD

**Issue by sunnyHelen** [NVlabs/geomapnet#30]
Do you ever try to train or test on KITTI dataset? Can I use your model to predict trajectory on other dataset except 7 Scenes and RobotCar?

**Comment by samarth-robo** [comment]
Hi @sunnyHelen , yes you can use MapNet for other datasets -- create your own dataloader by following the ones already provided. [Here](https://github.com/NVlabs/geomapnet/blob/master/dataset_loaders/README.md) is some documentation.

**Comment by sunnyHelen** [comment]
Nice, thanks a lot~

**Comment by sunnyHelen** [comment]
Hi, I created a data loader KITTI dataset. But when I visualize, I find even the trajectory of ground-truth are totally wrong. Like this:
![image](https://user-images.githubusercontent.com/28101548/71393453-bbefeb00-2660-11ea-9e95-c3fb24dcd6e6.png)
And here's the description of the ground-truth pose:
<img width="503" alt="image" src="https://user-images.githubusercontent.com/28101548/71393559-2d2f9e00-2661-11ea-9f36-d2b6c4ca61e7.png">
I just used this ground-truth for visualization. Is this wrong? Or do you know what may cause this result?

**Comment by sunnyHelen** [comment]
The right trajectory should be like this:
<img width="301" alt="image" src="https://user-images.githubusercontent.com/28101548/71393692-e1c9bf80-2661-11ea-9ca4-8c2541f90a21.png">

**Comment by samarth-robo** [comment]
Hi @sunnyHelen it is hard to debug this without looking at your dataloader code. It seems like you should be able to load the `poses` file directly into the `ps` variable in your dataloader. I suggest to first turn all normalization off by setting `mean_t=[0, 0, 0]`, `std_t=[1, 1, 1]`, `align_R = np.eye(3)`, `align_t = np.zeros(3)`, `align_s = 1`, and seeing if that produces a reasonable plot.
